src/config/nc.rs

```diff
@@ -10,7 +12,7 @@ pub(super) fn parse_config_file(
     let config = if glob { parse_glob(path) } else { parse(path) }?;
 
     Ok(PartialConfig {
-        database: Some(config.database.into()),
+        database: Some(AnyConnectOptions::from_str(&config.database.url())?),
```

Comment:
I think this is basically the same as
```suggestion
        database: Some(config.database.url().parse()?),
```
 I prefer the implicite use of `from_str` via parse - but totally up to you.